R/osmactive.R

```diff
@@ -213,7 +213,7 @@ get_cycling_network = function(
     ) |>
     # Remove poor quality smoothness:
     dplyr::filter(
-      ! smoothness %in% c("bad", "very_bad", "horrible", "very_horrible", "impassable")
+      ! (smoothness %in% c("bad", "very_bad", "horrible", "very_horrible", "impassable") & surface != "sett")
```

Comment:
Isn't the issue that the way is excluded because `smoothness == "bad"`? https://www.openstreetmap.org/way/4871777#map=19/55.959616/-3.194916&layers=N

A simpler way to include https://www.openstreetmap.org/way/4871777#map=19/55.959616/-3.194916&layers=N would be to simply remove links with bad smoothness, or to remove this filter alltogether.